core/arch/arm/crypto/ghash-ce-core_a64.S

```diff
@@ -17,6 +17,7 @@
 	.size func , .-func
 
 #define CPU_LE(x...)	x
+#define INC_HALF_CTR	0
```

Comment:
This deserves a comment I think. Not sure I get it. So we're incrementing the whole 128-bit value while upstream touches only the lower part? Why are we doing this?

Reply:
I'm adding:
``` C
/*
 * If the lower half of CTR is initialized with zeroes or a low value we
 * can expect that the upper half will remain unchanged. As an optimization
 * make the code to increase the upper half optional.
 */

Reply:
Totally clarifies the intent :+1: